tests/phpunit/tests/meta/updateMetadata.php

```diff
@@ -27,4 +27,71 @@ public function test_slashed_key_for_existing_metadata() {
 		$found = get_metadata( 'post', 123, 'foo\foo', true );
 		$this->assertSame( 'baz', $found );
 	}
+
+	/**
+	 * @ticket 54316
```

Comment:
```suggestion
	 * @ticket 54316
	 *
	 * @group user
	 *
	 * @covers ::clean_user_cache
	 *
	 * @global wpdb $wpdb WordPress database abstraction object.
```

:1234: This also applies to the docblock of the test below. I've used the ordering suggested by @SergeyBiryukov in [WP#56070](https://core.trac.wordpress.org/ticket/56070).